cmd/srcd/daemon/daemon.go

```diff
@@ -65,6 +65,18 @@ func Kill() error {
 	return nil
 }
 
+// CleanUp removes all resources created by daemon on host
+func CleanUp() error {
+	datadir, err := datadir()
+	if err != nil {
+		return err
+	}
+
+	gitbaseIndexDir := filepath.Join(datadir, "gitbase")
```

Comment:
I assume we're only storing there only the default config file, and `gitbase` indexes.
The problem with current implementation is that if I understood it properly, we're not ensuring now that `prune` will delete everything that Engine could create in host machine, but only the gitbase indexes (what is ok in terms of the issue, but not in terms of `prune` command)

I'd suggest to separate `config` and `data` dirs, so we could delete the whole `data` dir with `prune`, also all new things that could be created by Engine at some point.

If you agree with this suggestion, let me know and I'll open an issue with it.